lib/bundle/index.js

```diff
@@ -56,20 +56,33 @@ function crawl (parent, key, path, pathFromRoot, indirections, inventory, $refs,
       // which later determines which keys get dereferenced and which ones get remapped
       let keys = Object.keys(obj)
         .sort((a, b) => {
-          let aDefinitionsIndex = `${pathFromRoot}/${a}`.lastIndexOf(options.bundle.defaultRoot);
-          let bDefinitionsIndex = `${pathFromRoot}/${b}`.lastIndexOf(options.bundle.defaultRoot);
-          // Most people will expect references to be bundled into the the defaultRoot property,
-          // so we always crawl that property first, if it exists.
-
-          if (aDefinitionsIndex !== bDefinitionsIndex) {
-            // Give higher priority to the $ref that's closer to the "definitions" property
-            return bDefinitionsIndex - aDefinitionsIndex;
+
+
+          // performance optimization: indexOf is ~12 times faster than lastIndexOf as of now, LIO becomes a bottleneck in larger schemas
```

Comment:
hmmm, this might be tad too optimistic. I feel like this check may actually hamper the performance in other cases.

Apparently, $refs in Stripe doesn't match `options.bundle.defaultRoot`, but there are cases where `indexOf` is likely to be often `>= 0`, thus we'll end up calling both `indexOf` & `lastIndexOf`.
It simply depends on the particular input.

The default value we use is `#/definitions`, so if it's a OAS3 document that path is indeed never going to be hit, buuuut if it's OAS2 or an older JSON Schema one may use `#/definitions`, and the results might be different.
Yet again it depends on the scenario, because one may have no $refs under `#/definitions`.

In any case, feels like the best would be to make `defaultRoot` be null if one doesn't care about this output optimization.
You certainly don't, as this only affects the output document.
In other words, I'm going to expose a way to avoid this piece of code entirely.
We already have everything in place, just need a small tweak.